### ISSUE TITLE
Issue with defaults for inline styles

### ISSUE THREAD

**Issue by winkelsdorf** [tradle/rn-markdown#5]
With the current configuration every style used (e.g. **bold**, or [Links](Links)) is rendered on a newline before and after that style.

It's already visible on https://tradle.github.io/rn-markdown-playground/.

Example Markdown:
```markdown
Please see the following [link](https://www.google.com).
```

It is usually rendered like here on GH:
> Please see the following [link](https://www.google.com).

While using `rn-markdown` it renders like this:
> Please see the following
> [link](https://www.google.com)
> .

This seems to be a bit off the default markdown style, doesn't it? I am wondering if this is coming from your component or marked..

**Comment by mvayngrib** [comment]
@winkelsdorf you're right, i think this was broken recently! I restored to a previous version for now, if you'd like to work on compatibility with the latest react/react-native-web, happy to accept PRs :)

**Comment by winkelsdorf** [comment]
@mvayngrib I see, thanks for the confirmation! Sorry, won't be able to enhance it currently, tbh I'm mainly a Swift Dev who just needs to get something done in RN quickly. But I'll pin this, just in case.
